### DISCUSSION TITLE
yaml格式的配置如何使用是否有文档啊

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [apolloconfig/apollo#4444] [Q&A] [answered]
yaml格式的配置如何使用是否有文档啊，找了一圈没有发现有系统的文档啊。看到了这个https://github.com/apolloconfig/apollo/pull/1944 ，但是服务端怎么调整呢？服务端怎么从properties切换到yaml呢？

**Comment by nobodyiam** [comment, marked as answer]
Simply create a namespace with yaml format.

![image](https://user-images.githubusercontent.com/837658/177227376-18431608-07b4-4a9d-b8e3-a5a4b1b4b025.png)
